js/bootstrap-datepicker.js

```diff
@@ -919,8 +919,8 @@
 				tooltip;
 			if (isNaN(year) || isNaN(month))
 				return;
-			this.picker.find('.datepicker-days thead .datepicker-switch')
```

Comment:
My version is more concise. Which option is faster, I do not know. I go back as it was.